speedy-recovery-client/src/components/core/App.js

```diff
@@ -176,7 +176,7 @@ class App extends Component {
               console.error(error);
             });
         }
-        this.updateStateAppointment(user.id);
+        //this.updateStateAppointment(user.id);
```

Comment:
Why is this commented out? Can it be deleted?

Reply:
Remember i told you there was a bug in the dev branch. I dont know why the updateStateAppointment was called in this line. But yes we can delete it.